README.md

```diff
@@ -11,7 +11,7 @@ A Manual test of using the https://github.com/pixelhandler/ember-jsonapi-resourc
 
 ## Running / Development
 
-* `ember server` (an http-proxy is setup to use http://api.pixelhandler.com)
+* `ember server --proxy` (an http-proxy is setup to use api @ http://pixelhandler.com)
```

Comment:
@aars I'm not sure why, but I've been using just `ember server` and the proxy setup here https://github.com/pixelhandler/jr-test/blob/master/server/proxies/api.js just works.